pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -5,22 +5,22 @@ import { withGlobalStore } from 'store'
 
 const Overview = ({ store, ...props }) => (
   <OverviewTemplate
-    metadataCount={store.dataProvider?.statistics.metadataCount}
-    fullTextCount={store.dataProvider?.statistics.fullTextCount}
-    timeLagData={store.dataProvider?.depositDates.timeLagData}
+    metadataCount={store.dataProvider?.statistics?.metadataCount}
```

Comment:
you will need to change everywhere not just in overview template

Reply:
This is what I am afraid of

Reply:
The only Settings page is not covered. This is what I don't cate much but we should keep it in mind. Also, the Settings page is not moved to templates.